my-app/src/App.css

```diff
@@ -110,11 +115,11 @@ h2 {
 .us-page {
   display: flex;
   justify-content: center;
-  /*align-items: center;*/
   flex-direction: column;
   width: 800px;
-  border: 0.5px solid gray;
-  padding-left: 10px;
+  border: 2px outset #c3c3c375;
```

Comment:
It looks like this hex code is too long

Reply:
Its a hex code for color and transparency.

Reply:
🤯 oh wow, I learned something new!